README.md

```diff
@@ -69,6 +70,13 @@ Additional lists you might find useful:
 
 # Plugins
 
+## APM
+*Plugins for Application Performance Monitoring.*
+
+- [jippi/cakephp-newrelic](https://github.com/jippi/cakephp-newrelic/tree/cake3) - A complete plugin that enables full New Relic integration for a CakePHP application, including CLI naming, exceptions sending, custom timings, etc. It does not depends on New Relic agent.
+
+- [brunitto/cakephp-new-relic](https://github.com/brunitto/cakephp-new-relic) - A simple plugin that enables just name transaction and browser timing for a CakePHP 3 application using the New Relic PHP agent.
```

Comment:
The Syntax for Plugin Name looks wrong. Also should contain the word Plugin as Outlines in contribution.md